src/service/search/searchController.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+const router = express.Router();
+const searchService = require("./searchService");
+
+router.get("/search/:businessName", async (req: any, res: any) => {
```

Comment:
Query Param might work better here to avoid clashing. Eg (/search/:businessName, /search/:serviceName, /searchProductName)
Also, search gonna have to include location as well (That's my bad. I just realizing I didn't spec out the description properly)

Example search would be like /search?for=business&?in=nassau